kiva/api.py

```diff
@@ -1,3 +1,142 @@
+"""
+
+Affine Transformations
+======================
+
+- :func:`~.affine_identity`
+- :func:`~.affine_from_values`
+- :func:`~.affine_from_scale`
+- :func:`~.affine_from_rotation`
+- :func:`~.affine_from_translation`
+- :func:`~.scale`
+- :func:`~.rotate`
+- :func:`~.translate`
+- :func:`~.concat`
+- :func:`~.invert`
+- :func:`~.is_identity`
+- :func:`~.affine_params`
+- :func:`~.tsr_factor`
+- :func:`~.trs_factor`
+- :func:`~.transform_point`
+- :func:`~.transform_points`
+
+Constants
+---------
+
+- :attr:`~.IDENTITY` - represents the :func:`~.affine_identity` matrix.
+
+Drawing Utilities
```

Comment:
Nitpick: (also 🚲🏡) "Utilities" seems like it should include functions, but all of these are constants.